src/samplers/pmmh.jl

```diff
@@ -11,17 +11,17 @@ alg = PMMH(100, SMC(20, :v1), MH(1,:v2))
 alg = PMMH(100, SMC(20, :v1), MH(1,(:v2, (x) -> Normal(x, 1))))
 ```
 """
-mutable struct PMMH <: InferenceAlgorithm
+mutable struct PMMH{T, A} <: InferenceAlgorithm
```

Comment:
Consider `A <: Tuple`, to ensure the same type constraints are in place as before. I'm not sure if this is entirely necessary though.

Reply:
Sure.